app/views/monitoring.html

```diff
@@ -0,0 +1,360 @@
+              </h1>
+            </div>
+            <alerts alerts="alerts"></alerts>
+            <div class="data-toolbar">
+              <ui-select ng-model="kindSelector.selected" theme="bootstrap" search-enabled="true" ng-disabled="kindSelector.disabled" title="Choose a resource" class="data-toolbar-dropdown">
+                <ui-select-match placeholder="Choose a resource">{{$select.selected.label ? $select.selected.label : ($select.selected.kind | humanizeKind : true)}}</ui-select-match>
+                <ui-select-choices repeat="kind in kinds | filter : {kind: $select.search} : matchKind | orderBy : 'kind'">
+                  <div ng-bind-html="(kind.label ? kind.label : (kind.kind | humanizeKind : true)) | highlight: $select.search"></div>
+                </ui-select-choices>
+              </ui-select>
+              <div class="vertical-divider"></div>
+              <div class="data-toolbar-filter form-inline">
+                <div class="form-group filter-controls">
+                  <label for="events-filter" class="sr-only">Filter by name</label>
+                  <input type="search"
+                        placeholder="Filter by name"
+                        class="form-control"
+                        id="events-filter"
+                        ng-model="filters.text">
+                </div>
+              </div>
+              <div class="data-toolbar-checkbox">
+                <label>
+                  <input type="checkbox" ng-model="filters.hideOlderResources">Hide older resources
+                </label>
+              </div>
+            </div>
+          </div>
+        </div><!-- /middle-header-->
+        <div class="middle-content">
+          <div class="container-fluid">
+            <div class="row">
+              <div class="col-md-12">
+                <div ng-if="kindSelector.selected.kind === 'All' || kindSelector.selected.kind === 'Builds'">
+                  <h2>Builds</h2>
+                  <div class="list-view-pf">
+                    <div class="list-group-item" ng-if="!(filteredBuilds | hashSize)">
+                      <div class="list-view-pf-main-info">
+                        <em>
+                          <span ng-if="!(builds | hashSize)">
+                            There are no builds in this project.
+                          </span>
+                          <span ng-if="builds | hashSize">
+                            The current filters are hiding all builds.
+                          </span>
+                        </em>
+                      </div>
+                    </div>
+                    <div class="list-group-item list-group-item-expandable"
+                         ng-repeat-start="build in filteredBuilds track by (build | uid)"
+                         ng-click="expanded = !expanded; toggleItem(build, expanded)"
+                         style="cursor: pointer;"
```

Comment:
I'd add this style to the `.list-group-item-expandable` class instead of inline.